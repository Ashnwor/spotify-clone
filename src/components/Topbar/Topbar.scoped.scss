@import '../../styles/color-palette';

.topbar {
  display: flex;
  height: 60px;
  flex-direction: row;
  justify-content: space-between;
  padding: 16px 32px;
  background: #090909;
}

.nav-buttons {
  display: flex;
  gap: 16px;
  flex-direction: row;
}

.user-menu {
  align-self: center;
}
