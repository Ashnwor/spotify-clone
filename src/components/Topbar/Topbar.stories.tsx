import React from 'react';
import Topbar from './Topbar';

export default {
  title: 'Topbar',
};

export const topbar = () => <Topbar />;
