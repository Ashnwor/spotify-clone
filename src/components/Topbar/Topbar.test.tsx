import React from 'react';
import { shallow } from 'enzyme';
import Topbar from './Topbar';

it('runs correctly', () => {
  const wrapper = shallow(<Topbar />);
  expect(wrapper).toMatchSnapshot();
});
