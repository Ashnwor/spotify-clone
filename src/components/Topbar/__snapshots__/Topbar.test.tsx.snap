// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runs correctly 1`] = `
<div
  className="topbar"
>
  <div
    className="nav-buttons"
  >
    <TopbarNavBtn
      direction="left"
    />
    <TopbarNavBtn
      direction="right"
    />
  </div>
  <PillMenu
    Text="Unknown User"
    className="user-menu"
  />
</div>
`;
